be/src/exec/pipeline/fragment_context.cpp

```diff
@@ -317,4 +317,21 @@ void FragmentContext::count_down_epoch_pipeline(RuntimeState* state, size_t val)
     state->query_ctx()->stream_epoch_manager()->count_down_fragment_ctx(state, this);
 }
 
+void FragmentContext::init_jit_profile() {
+    if (runtime_state() && runtime_state()->is_jit_enabled() && runtime_state()->runtime_profile()) {
+        _jit_timer = ADD_TIMER(_runtime_state->runtime_profile(), "jit_cost_time");
+        _jit_counter = ADD_COUNTER(_runtime_state->runtime_profile(), "jit_counter", TUnit::UNIT);
```

Comment:
profile name should use camelCase instead of underscores to keep in line with the current style?